packages/venia-ui/lib/components/CheckoutPage/OrderConfirmationPage/orderConfirmationPage.js

```diff
@@ -1,34 +1,122 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { propType } from 'graphql-anywhere';
```

Comment:
Maybe rename this on import? Open to suggestions here, but I find this pretty vague.

```suggestion
import { propType as queryData } from 'graphql-anywhere';
```